### PR TITLE
SITES-606 Add feedback button on all headers

### PR DESCRIPTION
**Pull request by maxious**
Includes current page url from the current request object

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -28,8 +28,9 @@
             .brandbar.default
               .b1
               .b2
-          = #render partial: 'feedback_and_menu'
-
+          .feedback{tabindex:1}
+            %a.button--feedback{href:"https://qfrankp.au1.qualtrics.com/jfe/form/SV_5BgphbOHhi7qXe5?pageviewed="+request.url, role:"button"}
```
Comment (ltankey) [inline app/views/layouts/application.html.haml]: Could we use an environment variable, just in case it changes?

**Comment by ltankey** [comment]
Changelog?

**Comment by simonschwartz** [comment]
LGTM

**Comment by maxious** [comment]
rebase'd

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -28,8 +28,9 @@
             .brandbar.default
               .b1
               .b2
-          = #render partial: 'feedback_and_menu'
-
+          .feedback{tabindex:1}
+            %a.button--feedback{href:ENV.fetch('FEEDBACK_URL','')+request.url, role:"button"}
```
Comment (bjgordon) [inline app/views/layouts/application.html.haml]: There's no requirement given, but I wonder if FEEDBACK_URL is empty should we just not show it at all?

Comment (maxious) [inline app/views/layouts/application.html.haml]: Did consider but then we wouldn't be testing that code everytime.

Comment (bjgordon) [inline app/views/layouts/application.html.haml]: Hmm maybe this needs a test then?

**Comment by ltankey** [comment]
LGTM

**Pull request merged**